openapi-v2.yaml

```diff
@@ -2,63 +2,35 @@ openapi: 3.0.3
 info:
   title: dbt Cloud API v2
   version: 2.0.0
-  description: |
```

Comment:
Lmk if I'm using the spotlight tool wrong. Looked like the PR description link was out of date, as it still had your 'gideon' namespace in the servers list. Think it's currently failing: https://elements-demo.stoplight.io/?spec=https://github.com/dbt-labs/dbt-cloud-openapi-spec/blob/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

Reply:
@MitchellBarker you have to paste in the "raw" file like this: https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

Reply:
Here's an updated link. Putting it in the description too: https://elements-demo.stoplight.io/?spec=https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

Reply:
Duh, my bad haha